### PR TITLE
decrypt previous convo

### PR DESCRIPTION
**Pull request by chrisadubois**
# Pull Request Template

## Description
NOTE: Need to add tests
Allowing previous field to be decrypted to change the system message in web client.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

for this PR: https://sqbu-github.cisco.com/WebExSquared/webex-web-client/pull/392

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

NEED TO ADD TESTS

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js [hunk 1]
```diff
@@ -151,6 +154,10 @@ export const transforms = toArray('inbound', {
       promises = activity.children.map((child) => ctx.transform('decryptObject', keyUrl, child.activity));
     }
 
+    // if (activity.object && activity.object.previous) {
+
+    // }
+
```
Comment (stevem63) [inline packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js]: assuming you just forgot to delete

File: packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js [hunk 2]
```diff
@@ -126,6 +126,9 @@ export const transforms = toArray('inbound', {
     if (conversation.avatarEncryptionKeyUrl) {
       promises.push(ctx.transform('decryptObject', conversation.avatarEncryptionKeyUrl, conversation.avatar));
     }
+    if (conversation.previous) {
+      promises.push(ctx.transform('decryptPropDisplayName', usableKey, conversation.previous));
+    }
```
Comment (stevem63) [inline packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js]: I'm confused why we're doing this in decryptConversation and not decryptActivity

Comment (stevem63) [reply packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js]: ^^discussed offline

File: package-lock.json [hunk 3]
```diff
@@ -6929,7 +6929,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
```
Comment (adamweeks) [inline package-lock.json]: Package locks without package changes should not be committed. Please remove this change.

Comment (chrisadubois) [reply package-lock.json]: done

File: packages/node_modules/@webex/plugin-meetings/browsers.js [hunk 4]
```diff
@@ -86,7 +86,7 @@ module.exports = function createBrowsers() {
     //   }
     // }
     ChromeH264: {
-      base: 'ChromeHeadless',
+      base: 'Chrome',
```
[the remark this answers is not in the document]
Comment (taymoork2) [reply packages/node_modules/@webex/plugin-meetings/browsers.js]: Can you revert this back

[the remark this answers is not in the document]
Comment (taymoork2) [reply packages/node_modules/@webex/plugin-meetings/browsers.js]: Can you revert this back

**Pull request merged**